app/Http/Controllers/AdminDash.php

```diff
@@ -1045,6 +1046,127 @@ public function emailFeedback(Request $request, $id) {
         return redirect()->back()->with('success', 'The email has been sent to the pilot successfully.');
     }
 
+    public function showTrainerFeedback() {
+        $users = User::with('roles')->where('status', '1')->get();
+        $ins = $users->filter(function ($user) {
+            return $user->hasRole('ins');
+        });
+
+        $mtr = $users->filter(function ($user) {
+            return $user->hasRole('mtr');
+        });
+        $feedbackOptions = $ins->merge($mtr);
```

Comment:
This would be cleaner and more efficient with:

```php
$feedbackOptions = $users->filter(function ($user) {
  return $user->hasRole('ins') || $user->hasRole('mtr');
});
```

Reply:
Done